src/integrationCapture.ts

```diff
@@ -169,6 +189,21 @@ export default class IntegrationCapture {
         return this.applyProcessors(queryParams, getHref(), this.initialTimestamp);
     }
 
+    /**
+     * Captures local storage based on the integration ID mapping.
+     */
+    public captureLocalStorage(): Dictionary<string> {
+        let localStorageItems: Dictionary<string> = {};
+        for (const key in integrationMapping) {
+            const localStorageItem = localStorage.getItem(key);
+            if (localStorageItem) {
+                localStorageItems[key] = localStorageItem;
+            }
+        }
+
+        return this.applyProcessors(localStorageItems, getHref(), this.initialTimestamp);
```

Comment:
any reason we aren't including getHref(), and initialTimestamp in `captureCookies`?